test/scripts/run-e2e-tests.js

```diff
@@ -58,10 +58,10 @@ const setupTestEnvironment = async (generatedAppPath) => {
 yargs
   .parserConfiguration({
     /**
-     * This lets us pass any other arguments to playwright
-     * e.g. the name of a specific test or the project we want to run
+     * When unknown options is false, using -- to separate playwright args from test:e2e args works
+     * When it is true, the script gets confused about additional arguments, with or without using -- to separate commands
      */
-    'unknown-options-as-args': true,
+    'unknown-options-as-args': false,
```

Comment:
This and the corresponding update to the contributor doc should really be a separate PR, but without the ability to only run one file from one domain, it's extremely time consuming to test this, so I've just included it here.

Reply:
> but without the ability to only run one file from one domain

You can do this `yarn test:e2e --domains=admin -- transfer`

Reply:
no, that's what I mean. This has to be flipped to false or else that doesn't work, at least for me. The previous comment+value appears to have been a lie